backend/src/main/java/com/atlantbh/auction/repository/BaseRepository.java

```
@@ -0,0 +1,11 @@
+package com.atlantbh.auction.repository;
+
+import java.util.List;
+
+public interface BaseRepository<M, I> {
+    public void add(M entityClass);
+    public M update(M entityClass);
```

All these methods should throw a custom exception if reading/writting to database fails and it should be included here in signature.